prod.log

```diff
@@ -0,0 +1,12 @@
+time=1759153340266 level=info msg="initDb: database initialized"
```

Comment:
delete this file and add it to gitignore. All possible log files that may be generated. add it to gitignore.